test/widgets/action_sheet_test.dart

```diff
@@ -442,6 +445,44 @@ void main() {
             });
       });
 
+      testWidgets('on topic move, acts on new topic', (tester) async {
+        final stream = eg.stream();
+        const topic = 'old topic';
+        final message = eg.streamMessage(flags: [MessageFlag.read],
+          stream: stream, topic: topic);
+        await setupToMessageActionSheet(tester, message: message, narrow: TopicNarrow.ofMessage(message));
```

Comment:
nit: it might be good to wrap this line because `narrow` is particularly interesting to this test